openfisca_core/columns.py

```diff
@@ -218,6 +218,13 @@ class DateCol(Column):
     json_type = 'Date'
     val_type = 'date'
 
+    def __init__(self, default = None, **kwargs):
+        super(DateCol, self).__init__(**kwargs)
+        if default is None:
+            default = datetime.date.fromtimestamp(0)  # 0 == 1970-01-01
```

Comment:
I first used `date.min` but it seems that it breaks date variables like `birth` and perhaps more.

I could investigate and port them, but I thought that there is actually no concept of default date.

Should I disable completely the `default` value and handle each `DateCol` separately?
I can try that.